### DISCUSSION TITLE
Skip break

### DISCUSSION THREAD

**Discussion by LIII-Works** [ppy/osu#13078] [Ideas]
Kiddo Kris
-

> I know it is hard to read ton of words,so I did my best to make it easier to read.

**Describe the new feature:** 
-

> I would like to see this in osu!lazer,really.

Allow player to skip long break,a long break always interrupt focus

>A long Break
 ![Skip](https://user-images.githubusercontent.com/56265123/69006291-faaacb00-0967-11ea-80b2-49c89bb158fe.png)

**Proposal designs of the feature:**
-
Allow player to skip long break like skip the intro
- maybe higher or equal to 5 second should be allowed to skip

**Comment by bdach** [comment]
There's a stale PR that seemed to aim to implement this at #3264.
